game_assets.py

```diff
@@ -77,7 +77,7 @@
     "Nami": {"Gold": 2, "Board Size": 1},
     "Neeko": {"Gold": 4, "Board Size": 1},
     "Nidalee": {"Gold": 1, "Board Size": 1},
-    "Nunu": {"Gold": 3, "Board Size": 1},
+    "Nunu & Willump": {"Gold": 3, "Board Size": 1},
```

Comment:
Is this supposed to be Nunu & Willump? On here is says Nunu but I haven't been in game and seen it
https://app.mobalytics.gg/tft/champions/nunu

Reply:
yo nice code btw.
so your code had the "Nunu" and OCR didnt recognized the champion until i added the & Willump, idk if it is a language setting thing or smth but it works for me on English now